### DISCUSSION TITLE
Img rendering too slow....

### DISCUSSION THREAD

**Discussion by hpachy** [vercel/next.js#23303] [Help] [answered]
Hello everyone, 

I'm looking for some information can help me to understand how to optimize img's display :

-When I load a page with imgs all imgs take 2/3 second to be displayed after the page loaded
I use this code :
```js
  <Image
     alt={title} 
     // url passed like https://s3bucket/img.png
     //domain of the img declared in the next.config
     src={img}
     layout="fill"
     objectFit="cover"
     quality={100}
   />
```
I try to reduce the quality but nothing changed....

I wonder what can i do to avoid the loading time but I found nothing or most probably I miss understood something so I'm asking for help...

**Comment by akellbl4** [comment, marked as answer]
Hi,
What is the size of your images? Because when you use `layout="fill"` and `objectFit="cover"` Next.js don't do any scaling of images. 

Also, It could be a bottleneck in between your Next.js app and your s3 bucket. Do you have that lag if you connect images without `next/image` just like `<img src={img} loading="lazy"/>`

**Reply by hpachy** [reply]
Hi !

I don't really know the size of my img sadly... that's the problem and it was the only way to have a pleasant display...
I just tried with classic img tag but even if the loading look shorter It's still not natural. You can feel the display processed as 2 frames / times

**Reply by akellbl4** [reply]
Hmm, I was thinking about it and realized that the only problem that you can have with `next/image` is the long loading time.
Try to compare that time between classic `<img/>` and `next/image`. You should have a bit slower loading for an image with `next/image` only on first loading because then the image will be cached and loading time should be the same or faster than classic `<img/>`

Your experience with slow images could be caused by `loading="lazy"` and you can try `loading="eager"` for images that are in view on the first screen of your website. Technic with lazy loading is used for every `next/image` because many images render outside of the first screen and that images will be rendered only when the user scrolls to that place.

**Reply by hpachy** [reply]
Yes I see, you totally right but the big surprise is even if I add the properties 'priority' and 'eager' to the <Image /> component it's still the same... an loading time of 1/2 second after content loaded

**Reply by akellbl4** [reply]
Only one suppose that I have all about the size of the images. Try to use an image with `width` and `height` and property `objectFilt="cover"` if you want to scale your image proportionally. It should make your images smaller and they should be loaded faster.

**Reply by hpachy** [reply]
It's not more faster but I do not have CLS anymore, I think my 'problem' is totally normal after all because it's the first generation of my page so I think it's normal to the first generation/ rendering img takes some time ! 

thx @akellbl4  I learned a lot thank to you !

**Reply by mraxime** [reply]
@hpachy I'm not sure but the problem can also be related to this issue: #23637 .
Did you try with version 10.0.7 ?
